frontend/src/components/pages/admin/DisplayAssessmentsPage.tsx

```diff
@@ -162,8 +183,44 @@ const DisplayAssessmentsPage = (): React.ReactElement => {
     return filteredTests?.map(getAssessments);
   }, [search]);
 
+  const filteredAssessements = React.useMemo(() => {
+    let filteredTests: AssessmentTypes[] = searchedAssessements as AssessmentTypes[];
+
+    if (grades.length !== 0) {
```

Comment:
is there a way you can condense these into one filter?

Reply:
Changed!